packages/core/src/sass/app/_common/components/app-notification-message.scss

```
@@ -122,10 +121,11 @@
         margin-bottom: 16px;
     }
     @include mobile {
-        max-width: 348px;
+        max-width: 328px;
```

Same here, you can do:
```suggestion
        max-width: calc(100% - 0.8rem);
        margin-right: 0.4rem;
```
So we wouldn't need the additional screen media query for `320px` on L84.